src/services/CampaignService.js

```diff
@@ -81,6 +81,13 @@ class CampaignService {
           $skip,
         },
       })
+      .then(resp => ({
+        ...resp,
+        data: resp.data.filter(
+          milestone =>
+            !(milestone.donationCounters.length <= 0 && milestone.status === Milestone.COMPLETED),
+        ),
+      }))
```

Comment:
I tried [features query](https://docs.feathersjs.com/api/databases/querying.html), but not succeed.  
Can we make this as a workaround? Or could you give some help about featuers query?